src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.test.tsx

```diff
@@ -75,9 +143,33 @@ it("shows the premium upgrade cta if the user is not a premium subscriber", () =
     name: "Upgrade to ⁨Premium⁩",
   });
   expect(premiumCtas.length).toBe(2);
+
+  await user.click(premiumCtas[0]);
+
+  const productTabYearly1 = screen.queryByRole("tab", { name: "Yearly" });
```

Comment:
Question pending with content …

Reply:
… since the strings are pending and the current design says yearly I’ll go ahead and merge as is. If we decide to change the wording addressing the change in a follow-up is very low effort.